ComparisonWithOrleans.md

```diff
@@ -0,0 +1,203 @@
+
+#### Automatic Actor Placement and Load Balancing
+
+  * Orleans Grains are deployed on silos that can span multiple cluster nodes and the user has no influence on their precise placement or load-dependent movement. This means that making use of elastically provisioned resources is fully automatic and built into the model.
+  
+  * Elastic deployment and load balancing are features that Akka users opt into by using ClusterSharding or cluster-aware routers with remote deployment. Otherwise Actors are created explicitly on a given node (this can be influenced using the configuration file in order to allow deployment decisions to be taken by the operations personnel and not the programmer).
+  
+### Programming Model
+
+#### Virtual Actors
+
+  * Orleans Grains are identified by their nominal type and a 128-bit GUID.
+  
+  * Akka Actors are identified by their ActorRef, which consists of the ActorPath (including physical network location and the names of all its ancestors) and its UID (a 32-bit integer that disambiguates different incarnations on the same ActorPath).
+
+Actors as well as Grains do not share memory with their peers, both only communicate via message-passing.
+
+##### Perpetual Existence
+
+discussed above
+
+##### Automatic Instantiation
+
+  * An Orleans Grain may at any given time have one or more activations (active instances) depending on deployment mode. This implies that stateful Grains will have to consider that there may be copies running outside of their control that are responsible for the same piece of data or functionality if automatic scale-out is enabled (see below).
+  
+  * An Akka ActorRef refers to an Actor that must have been previously instantiated (otherwise there would not be an ActorRef but only an ActorPath available) and that may have already terminated. There will never be two Actors running with the same ActorRef. When using ClusterSharding there can be exactly zero or one instances currently running for each logical Actor name at any given point in time (unless the user explicitly configures the cluster to tolerate operation under split-brain scenarios).
+
+see also above
+
+##### Location Transparency
+
+  * An Orleans Grain does not have knowledge of its physical location which can also transparently change during the runtime of the system. Users of a Grain’s function do not need to know the location of the Grain activation they are talking to. If my understanding is right then Grain handles can also be sent as part of messages.
```

Comment:
Yes, grain handles (we call them Grain references) can be sent or persistent. They are purely logical concept.